app/routes.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Route, IndexRoute } from 'react-router';
+import App from './containers/App';
+import MainPage from './containers/MainPage';
+import Login from './components/Login';
+import Loading from './components/Loading';
+import SettingsPage from './containers/SettingsPage';
+import ClinicUserSelectPage from './containers/ClinicUserSelectPage';
+import ClinicUserEditPage from './containers/ClinicUserEditPage';
+import NoUploadTargetsPage from './containers/NoUploadTargetsPage';
+import api from '../lib/core/api';
+
+export default (
+  <Route path="/" component={App} api={api}>
+    <IndexRoute component={Loading} />
+    <Route path="login" component={Login}/>
```

Comment:
I'm on the fence on this one. Currently, yes, they could be replaced by the paths in the pagesMap, but if we intended to make more complex paths in the future (regex-style pattern matching), then it's likely that those wouldn't be able to be flat references in the pagesMap. Perhaps that's premature optimization though and can be changed if we ever get there.